packages/node/src/sdk/index.ts

```diff
@@ -98,14 +98,14 @@ function shouldAddPerformanceIntegrations(options: Options): boolean {
 /**
  * Initialize Sentry for Node.
  */
-export function init(options: NodeOptions | undefined = {}): void {
+export function init(options: NodeOptions | undefined = {}): NodeClient {
```

Comment:
m: Although we always return a top-level client here, making this `NodeClient | undefined` gives us more flexibility in the future for top-level API without making breaking changes. It also allows us to align the type signature everywhere.

Reply:
Yes you're right, will update that!